build.gradle

```diff
@@ -296,8 +299,8 @@ publishing {
             groupId 'io.realm'
             artifactId = 'realm-android'
             artifact file("${buildDir}/outputs/gradle/realm-android-${currentVersion}.jar")
-            artifact file("${rootDir}/realm/realm-library/build/libs/realm-android-${currentVersion}-sources.jar")
-            artifact file("${rootDir}/realm/realm-library/build/libs/realm-android-${currentVersion}-javadoc.jar")
+            //artifact file("${rootDir}/realm/realm-library/build/libs/realm-android-${currentVersion}-sources.jar")
```

Comment:
Github issue... these changes are not in this PR... and this one has been closed. #1816 instead.